assets/dev/js/editor/controls/media.js

```diff
@@ -243,6 +251,54 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 	onBeforeDestroy() {
 		this.$el.remove();
 	},
+
+	async onMediaImageSize() {
+		this.ui.mediaImageSize.css( 'color', 'inherit' );
+		const previewPlaceHolder = this.$el.find( '.elementor-control-media__preview' );
+		const selectedImageId = {
+			id: this?.getControlValue( 'id' ) || previewPlaceHolder[ 0 ].getAttribute( 'data-id' ),
+			alt: this?.getControlValue( 'alt' ) || previewPlaceHolder[ 0 ].getAttribute( 'alt' ),
+		};
+
+		const selectedSize = this.ui.mediaImageSize.val();
+		let imageURL = await elementor.imagesManager.getImageUrl( {
+			id: selectedImageId.id,
+			size: selectedSize,
+		} );
+		const stateOptions = {
+			url: null,
+			id: selectedImageId.id,
+			alt: selectedImageId.alt,
+			dimensions: selectedSize,
+			source: 'library',
+		};
+		if ( imageURL ) {
+			stateOptions.url = imageURL;
+			this.setValue( stateOptions );
+		} else {
+			elementor.channels.editor.once( 'imagesManager:detailsReceived', ( data ) => {
+				imageURL = data[ selectedImageId.id ][ selectedSize ];
+				if ( imageURL ) {
+					stateOptions.url = imageURL;
+					this.setValue( stateOptions );
+				}
+			} );
+		}
+	},
+
+	toggleSizeControl() {
+		const previewImageExist = this.$el.find( '.elementor-control-media__preview' )[ 0 ]?.style?.backgroundImage;
+		const backgroundControl = this.$el[ 0 ]?.className?.match( /background_image/g );
```

Comment:
Maybe use `classList.contains()`?

Reply:
`classList` is no good for me. I need to test if there is any class which contain sub string of 

`xxxxxx-background_image`

Reply:
This is the only indicator that I found.